template-xml/app/src/main/java/co/nimblehq/template/xml/di/modules/StorageModule.kt

```diff
@@ -17,18 +14,8 @@ abstract class StorageModule {
 
     companion object {
 
-        @Provides
-        fun provideDefaultSharedPreferences(@ApplicationContext context: Context): SharedPreferences =
-            PreferenceManager.getDefaultSharedPreferences(context)
-
         @Provides
         fun provideSecuredLocalStorage(@ApplicationContext context: Context) =
             EncryptedSharedPreferences(context)
-
-        @Provides
-        fun provideNormalLocalStorage(
-            @ApplicationContext context: Context,
-            defaultSharedPreferences: SharedPreferences
-        ) = NormalSharedPreferences(context, defaultSharedPreferences)
```

Comment:
I think we forgot to remove deprecated NormalSharedPreferences

Reply:
Indeed. Thanks @luongvo